release.json

```
@@ -122,8 +122,8 @@
         },
         {
             "name": "gravitee-policy-jwt",
-            "version": "1.20.0",
-            "since": "3.10.0"
+            "version": "1.20.0-SNAPSHOT",
+            "since": "3.13.0"
```

```suggestion
            "since": "3.10.0"
```

you d'ont have to modify this, it's for CI purpose only

---

fixed